src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Stylesheets with a .scss extension cannot be linked directly in HTML. You need to compile them to CSS and then link the compiled .css file instead.